ci/jenkins/test.sh

```diff
@@ -590,12 +599,18 @@ function deliver_antrea {
             scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" flow-aggregator.tar jenkins@[${IP}]:${DEFAULT_WORKDIR}/flow-aggregator.tar
             ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n jenkins@${IP} "${CLEAN_STALE_IMAGES_CONTAINERD}; ${PRINT_CONTAINERD_STATUS}; ctr -n=k8s.io images import ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; ctr -n=k8s.io images import ${DEFAULT_WORKDIR}/flow-aggregator.tar" || true
         done
-    elif [[ $TESTBED_TYPE == "kind" ]]; then
+    elif [[ $TESTBED_TYPE == "kind" || $TESTBED_TYPE == "kind-flexible-ipam" ]]; then
             kind load docker-image antrea/antrea-agent-ubuntu:latest --name ${KIND_CLUSTER}
             kind load docker-image antrea/antrea-controller-ubuntu:latest --name ${KIND_CLUSTER}
             kind load docker-image antrea/flow-aggregator:latest --name ${KIND_CLUSTER} 
             kubectl config use-context kind-${KIND_CLUSTER}
-            docker cp ./build/yamls/antrea.yml ${KIND_CLUSTER}-control-plane:/root/antrea.yml 
+            docker cp ./build/yamls/antrea.yml ${KIND_CLUSTER}-control-plane:/root/antrea.yml
+            if [[ $TESTBED_TYPE == "kind-flexible-ipam" ]]; then
+                 kubectl get nodes -o wide --no-headers=true |  awk '{print $1}' | while read nodes; do
+                   configure_kind_ipam_routes "${nodes}"
+                 done
+                 FLEXIBLE_IPAM=true
```

Comment:
Yes, in test.sh  before running actual test it will run clean_antrea function and there I added on if condition to delete added ip routes, if there are no route then it will show the error:
### if [[ $TESTBED_TYPE == "kind-flexible-ipam" && $FLEXIBLE_IPAM == "true" ]]; then
     cidrs=($(kubectl get nodes -o jsonpath='{.items[*].spec.podCIDR}')) 
     for cidr in "${cidrs[@]}"; do 
       sudo ip route del $cidr 
     done
  fi